lessons/Data-Science/measures-of-center/langs/en-us/pages/histograms-moc.adoc

```diff
@@ -0,0 +1,77 @@
+= Histograms and Measures of Center
+
+@n The two histograms below represent the amount of money spent on a pair of jeans, one for a sample of sixth grade students and one for a sample of eighth grade students.
```

Comment:
Ambiguous. Which one is for 6th and which is for 8th?

More importantly, neither one is for boys or girls, so the question that follows doesn't make sense

Reply:
oopsies. I decided to remove "boys" and "girls" but then failed to do so. Will fix.